README.md

```diff
@@ -97,8 +97,14 @@ Options:
 Starting with v3.1 you can now use different ways of configuring it:
 
 - `lint-staged` object in your `package.json`
-- `.lintstagedrc` file in JSON or YML format
+- `.lintstagedrc` file in JSON or YML format, or you can be explicit with the extension:
+  - `.lintstagedrc.json`
+  - `.lintstagedrc.yaml`
+  - `.lintstagedrc.yml`
+  - `.lintstagedrc.js`
+  - `.lintstagedrc.cjs`
```

Comment:
Since these are JS files, do they belong in this list of the one below?

Reply:
I think that would be fine. I put then in the list since they were grouped with the other `rc` files